aion/web/elm/src/Update.elm

```diff
@@ -121,32 +140,32 @@ update msg model =
             else
                 model ! []
 
-        NoOperation ->
-            model ! []
-
-        SetNewQuestionContent questionContent ->
+        UpdateCreateQuestionForm name value ->
             let
                 oldPanelData =
                     model.panelData
             in
-                { model | panelData = { oldPanelData | newQuestionContent = questionContent } } ! []
+                { model
+                    | panelData =
+                        { oldPanelData | createQuestionForm = Forms.updateFormInput oldPanelData.createQuestionForm name value }
```

Comment:
Same here, you get rid of the dots using improved imports or cool variable bindings in **let** area just above and you're left with `createQuestionForm = updateFormInput createQuestionForm name value`. Seems more legible. I also think the naming could be better here. What if we changed **createQuestionForm** to **questionForm**? The **create** verb confuses me a lot as I imagine it is a name of a function that actually creates a questionForm